psh/pshapp/pshapp.c

```diff
@@ -521,6 +534,26 @@ static int psh_keyCode(const char *buff, int *pEsc)
 }
 
 
+static int psh_skipword(const char *str, int n, int iswordchr(int))
+{
+	const char *cur = str;
+	const char *end = str + n;
+	int step = (n < 0) ? -1 : 1;
+
+	/* If starting on a delimiter, skip it along with the contiguous delimiters */
+	while (cur != end && !iswordchr(*cur)) {
+		cur += step;
+	}
+
+	/* Find the end of the word */
+	while (cur != end && iswordchr(*cur)) {
+		cur += step;
+	}
+
+	return (n < 0) ? (str - cur) : (cur - str);
```

Comment:
I would be a bit more readable in my opinion.

```suggestion
        skipped = (n < 0) ? (str - cur) : (cur - str);
	return skipped;
```